cg-spec/editors_draft.html

```diff
@@ -2690,16 +2715,20 @@ <h2>Referential opacity</h2>
         <!--
         #### under OWL-entailment
 
-        :superman owl:sameAs :clark.
-        << :superman :can :fly >> :reportedBy :clark.
+        :superman owl:sameAs :clarkKent.
+        << :superman :can :fly >> :reportedBy :loisLane.
 
         #### does NOT entail
 
-        << :clark :can :fly >> :reportedBy :clark.
+        << :clarkKent :can :fly >> :reportedBy :loisLane.
         -->
         </pre>
 
-        <p>This is well suited for use-cases where the syntactical form of the annotated statements may be significant (like <a href="../UCR/rdf-star-ucr.html#annotate-statements-from-other-named-graphs-and-datasets">annotating statements from other graphs</a> or <a href="../UCR/rdf-star-ucr.html#annotate-commit-deltas">annotating commit deltas</a>). On the other hand, other use-cases would expect some degree of referential transparency. Consider the case of <a href="../UCR/rdf-star-ucr.html#meta-properties-over-properties">attributed/evidenced triples</a>. The annotation is understood to be about the fact asserted by the triple rather than the triple itself. As illustrated in the example below, one might then expect more inferences than provided by the current semantics.</p>
+      <p>This is well suited for use-cases where the syntactical form of the annotated statements may be significant, like <a href="../UCR/rdf-star-ucr.html#annotate-commit-deltas">annotating commit deltas</a> or differentiating the intended usage of terms from what they actually mean (like when documenting different world views or contexts of limited knowledge).</p>
+
+      <p class="issue" data-number="">In the Lotico presentation Pierre-Antoine explained the virtues of referential opacity by a more involved example, also using the Superman problem but extending it to an Explainable AI usecase (<a href="http://www.lotico.com/index.php/Metadata_for_RDF_Statements:_The_RDF-star_Approach">video at 1:08:12</a>, <a href="https://w3c.github.io/rdf-star/presentations/RDF-star_Lotico.pdf">slide 45</a>). I would find it advantageous if that example would also be added to this report as it seems to illustrate nicely what really drives the proposed semantics. It is another question if one thinks this works out as intended - obviously, I don't - but it gives people a much better base to understand and evaluate this proposal than the rather obscure Superman problem alone. Also, the Lotico example resorts to a hack, implementing surrogate N3 formulas via RDF lists of embedded statements, that IMO quite tellingly shows both the direction and the limitation of the proposed semantics. Again, obviously I don't believe in this design but I think readers should be enabled to form their own opinion.</p>
```

Comment:
If you find both examples equally unconvincing, I don't understand your motivation for changing one for the other...

Reply:
This level of commentary/discussion is not suitable for a PR nor a published document. Each of these `class="issue"` callouts should really refer out to their *own* issue (or PR, if substantially smaller than this), and not try to bring the whole debate inline in the report.

Reply:
> If you find both examples equally unconvincing, I don't understand your motivation for changing one for the other...

Well maybe I'm a bit harsh on the Superman problem but it can be hard to abstract away from a Comic scenario to the general problem of documenting world views. OTOH I found the Explainable AI use case quite interesting and also more rich on facets like the listing of embedded terms to emulate formulas. But this may be just me and I was just making a proposal.

@TallTed 
> This level of commentary/discussion is not suitable for a PR nor a published document. Each of these `class="issue"` callouts should really refer out to their _own_ issue (or PR, if substantially smaller than this), and not try to bring the whole debate inline in the report.

I apologize for the inconvinience but sometimes things are messy and it can't really be avoided. I can't discuss the setting and tone of a whole document in atomic PR's. I agree that this is not an optimal place to discuss details but I'm very grateful for the detailed critiques above and below nonetheless. I'll be happy to incorporate them and also split this PR once I have the feeling that we are on the same boat and/or if I see the benefit of smaller PRs to faciltate the discussion.